k8s/prod/common/ccpay/fees-register-api.yaml

```diff
@@ -0,0 +1,29 @@
+---
```

Comment:
this isn't approved for prod yet, please get it deployed to AAT and then we can get it reviewed by SecOps

Reply:
for timebeing removed config for production